engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/CreateGameScreen.java

```diff
@@ -138,10 +139,20 @@ public void set(Module value) {
         gameplay.setOptionRenderer(new StringTextRenderer<Module>() {
             @Override
             public String getString(Module value) {
-                return value.getMetadata().getDisplayName().value();
+            	return value.getMetadata().getDisplayName().value();
+            }
+            
+            @Override
+            public void draw(Module value, Canvas canvas)
+            {
+        		canvas.getCurrentStyle().setTextColor(
+        			validateModuleDependencies(value.getId()) 
+        				? Color.WHITE : Color.RED);
+            	super.draw(value, canvas);
+            	canvas.getCurrentStyle().setTextColor(Color.WHITE);
             }
         });
-
+        
```

Comment:
Can we please remove the space on this line (and on all others like it)?

Which IDE/text editor do you use? There should be a "trim trailing spaces" or something similar that you can enable in most softwares, to prevent this.